index.js

```diff
@@ -138,47 +175,58 @@ function nowServing(line) {
 const line = []
 
 // ***** Scope & Closures - Question 1 *****
-
-
+let lastTicket = 1
+function takeATicketNumber(line){
+  line.push(lastTicket)
+  lastTicket +=1
+  return `Welcome. You are ticket number ${(lastTicket-1)}`
+}
```

Comment:
Good job. However, instead of starting lastTicket at 1, let's start it at 0 and then swap lines 180 and 181, so the ticket gets incremented prior to being added to the line. We can also remove the -1 from the interpolation on 182